spec/support/user_helpers.rb

```diff
@@ -1,11 +1,14 @@
 module Helpers
   module UserHelpers
-    def login_admin
-      login(admin: true)
+    def login_admin(provider: 'MMT_2', providers: 'MMT_2')
```

Comment:
I kind of like the idea of having a `def real_login`. I think it could be a separate method, or it could call `login` passing in `real_login: true`, similar to how you originally made this `login_admin` method that calls `login`.